src/mixins/DrawingTool.js

```diff
@@ -3,6 +3,14 @@ import { types } from "mobx-state-tree";
 import Utils from "../utils";
 import throttle from "lodash.throttle";
 import { MIN_SIZE } from "../tools/Base";
+import { Hotkey } from "../core/Hotkey";
+
+const initializeHotkeys = (self) =>{
+  const hotkeys = Hotkey("Polygons");
+
+  hotkeys.addNamed("polygon:undo", () => self.getCurrentArea()?.undoRedoPoints(true));
+  hotkeys.addNamed("polygon:redo", () => self.getCurrentArea()?.undoRedoPoints(false));
```

Comment:
you should always remove hotkeys on destory, [check Tool.js](https://github.com/heartexlabs/label-studio-frontend/blob/master/src/components/Toolbar/Tool.js#L82-L100) for example or search for `hotkey.` usages in the project.